### ISSUE TITLE
Provide value when validation fails.

### ISSUE THREAD

**Issue by danlo** [rmosolgo/graphql-ruby#4367]
**Is your feature request related to a problem? Please describe.**

I was changing the error message for an inclusion validation from

```
argument :status, String, 'open/closed', required: false, validates: { inclusion: { in: %w[ open closed ], message: 'Validation-Error: Invalid value provided for %{validated} - %{value}'} }
```

Providing the value in question would help me see what the value that failed validation was.

**Describe the solution you'd like**

https://github.com/rmosolgo/graphql-ruby/blob/master/lib/graphql/schema/validator.rb#L136

Change: 
https://github.com/rmosolgo/graphql-ruby/blob/master/lib/graphql/schema/validator.rb#L136
```
interpolation_vars = { validated: validated.graphql_name }
```
to
```
interpolation_vars = { validated: validated.graphql_name, value: value }
```

**Describe alternatives you've considered**

I thought over monkey patching it, but I thought this would be useful to other people in the broader community context.

**Additional context**

None

**Comment by rmosolgo** [comment]
Agreed that would be nice, thanks for the suggestion! Do you want to take a crack at adding it to the gem?

**Comment by danlo** [comment]
Sorry, I haven't responded.  A child brought home an illness from school and it's been circulating.  I will try and create a PR for this it may be a while if someone else wants to pick it up :).
